packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -1357,17 +1358,20 @@ export default class LocusInfo extends EventsScope {
 
   /**
    * handles when the locus.self is updated
-   * @param {Object} self the locus.mediaShares property
-   * @param {Array} participants the locus.participants property
+   * @param {Object} self the new locus.self
    * @returns {undefined}
    * @memberof LocusInfo
    * emits internal events self_admitted_guest, self_unadmitted_guest, locus_info_update_self
    */
-  updateSelf(self: any, participants: Array<any>) {
-    // @ts-ignore - check where this.self come from
-    if (self && !isEqual(this.self, self)) {
+  updateSelf(self: any) {
```

Comment:
I feel like we should add a comment about locus.participants

and the delta changes

Reply:
done, added comment inside updateSelf()